src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1411,7 +1411,14 @@ BoundExpression bindDefaultArgument(SyntaxNode syntax, ParameterSymbol parameter
                     {
                         GenerateImplicitConversionError(diagnostics, syntax, conversion, defaultValue, parameterType);
                     }
-                    defaultValue = CreateConversion(defaultValue, conversion, parameterType, diagnostics);
+                    defaultValue = CreateConversion(
+                        defaultValue.Syntax,
+                        defaultValue,
+                        conversion,
+                        isCast: conversion.IsExplicit,
```

Comment:
From our offline discussion: is it possible to observe this change?

Reply:
It does not appear to be observable through either public API or through differences in diagnostics.

Reply:
Oddly enough I think the IsImplicit property on the IConversionOperation ends up being true if either the node is compiler generated, or if the conversion is not from a cast

https://github.com/dotnet/roslyn/blob/7e8b9ee7491b4d51809f5cc0273883692807c279/src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs#L952

Reply:
I made a pass over the usages of 'BoundConversion.IsExplicitCastInCode' and I wasn't able to come up with a scenario where behavior for a default argument would change.